docs/howto/openstack/nova/boot-image-volume.md

```diff
@@ -0,0 +1,395 @@
+# Converting boot-from-image servers to boot-from-volume
+
+You can freely move any server [between {{brand}}
+regions](move-server-between-regions), provided it uses
+boot-from-volume. Since such servers offer much more flexibility than
+those using boot-from-image, we recommend the former over the latter.
+You may still have boot-from-image servers, though. If you decide to
+move any of those between regions, you will discover that you cannot do
+so. The solution is first to convert them to boot-from-volume servers
+and then move them. In the following, we show how to perform the
+conversion using the {{gui}} or OpenStack CLI.
+
+## Preparation
+
+=== "{{gui}}"
+    Fire up your favorite web browser and navigate to the
+    [{{gui}}](https://{{gui_domain}}) start page. Log into your {{brand}}
+    account if you have to.
+=== "OpenStack CLI"
+    To work with the OpenStack CLI, make sure to properly [enable
+    it](/howto/getting-started/enable-openstack-cli) for the region your
+    boot-from-image server resides in.
+
+## Shutting down the server
+
+=== "{{gui}}"
+    Make sure the left-hand side vertical pane is in full view, then select
+    Compute → [Servers](https://{{gui_domain}}/compute/servers). You will
+    see all supported {{brand}} regions in the main pane. Expand the one
+    with the boot-from-image server you want to convert to boot-from-volume.
+    Click the orange :fontawesome-ellipsis-circle: icon at the right of the
```

Comment:
`:fontawesome-ellipsis-circle:` is not one of [shortcodes](https://squidfunk.github.io/mkdocs-material/reference/icons-emojis/) that mkdocs-material supports.

Reply:
Aha, good to have that reference! Since the icon I want is not there, I am falling back to the three-dots-in-a-circle thing.